vault/resource_aws_secret_backend.go

```diff
@@ -99,31 +99,34 @@ func awsSecretBackendResource() *schema.Resource {
 	}
 }
 
-func mountMigrationCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
-	if !diff.HasChange("path") {
+func mountMigrationCustomizeDiff_FieldPath(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	return mountMigrationHelper(ctx, diff, meta, consts.FieldPath)
+}
+
+func mountMigrationHelper(_ context.Context, diff *schema.ResourceDiff, meta interface{}, mountField string) error {
+	if !diff.HasChange(mountField) {
 		return nil
 	}
 
-	o, _ := diff.GetChange("path")
+	o, _ := diff.GetChange(mountField)
 	if o == "" {
 		return nil
 	}
 
 	// Mount Migration is only available for versions >= 1.10
-	client, e := provider.GetClient(diff, meta)
-	if e != nil {
-		return e
+	minVersion, err := version.NewVersion(consts.VaultVersion10)
```

Comment:
Can we pull these out to to `meta.go` instead? The versions are a known quantity.

We can add an `init()`  that sets some global variables up for us:

```go

var VaultVersion110

func init() {
    VaultVersion110 = version.Must(version.NewSemVer(consts.VaultVersion10))
}
```

Reply:
Added new variables and an `init` method